### PR TITLE
Added logs

### PR DESCRIPTION
**Pull request by VLiamin**

### CODE REVIEW

File: src/ProjectService.Business/Commands/Project/FindProjectsCommand.cs [hunk 1]
```diff
@@ -71,6 +73,8 @@ private async Task<List<DepartmentData>> GetDepartmentsThroughBroker(List<Guid>
 
         if (response.Message.IsSuccess)
         {
+          _logger.LogInformation("Departments were taken from the service.");
```
Comment (MariyaSin) [inline src/ProjectService.Business/Commands/Project/FindProjectsCommand.cs:76]: Maybe "Departments were taken from the Company Service."?

Comment (VLiamin) [reply src/ProjectService.Business/Commands/Project/FindProjectsCommand.cs]: Done

**Review by MariyaSin — APPROVED** [review]

**Review by BelehovEgor — APPROVED** [review]

**Review by ndoriomedov — APPROVED** [review]

**Review by Glunky — APPROVED** [review]

File: src/ProjectService.Business/Commands/Project/GetProjectCommand.cs [hunk 2]
```diff
@@ -56,6 +56,8 @@ private async Task<DepartmentData> GetDepartment(Guid departmentId, List<string>
 
       if (departmentFromCache.HasValue)
       {
+        _logger.LogInformation("Department was taken from the cache. Department's id: {departmentId}", string.Join(", ", departmentId));
```
Comment (MariyaSin) [inline src/ProjectService.Business/Commands/Project/GetProjectCommand.cs:59]: not departmentS

File: src/ProjectService.Business/Commands/Project/GetProjectCommand.cs [hunk 3]
```diff
@@ -72,6 +74,8 @@ private async Task<DepartmentData> GetDepartmentThroughBroker(Guid departmentId,
 
         if (departmentResponse.Message.IsSuccess)
         {
+          _logger.LogInformation("Department was taken from the service. Department's id: {departmentId}", string.Join(", ", departmentId));
```
Comment (MariyaSin) [inline src/ProjectService.Business/Commands/Project/GetProjectCommand.cs:77]: one department

File: src/ProjectService.Business/Commands/Task/GetTaskCommand.cs [hunk 4]
```diff
@@ -66,12 +68,14 @@ private async Task<DepartmentData> GetDepartmentThroughBroker(Guid authorId, Lis
...
       try
       {
-        Response<IOperationResult<IGetCompanyEmployeesResponse>> response = 
+        Response<IOperationResult<IGetCompanyEmployeesResponse>> response =
           await _rcGetCompanyEmployee.GetResponse<IOperationResult<IGetCompanyEmployeesResponse>>(
             IGetCompanyEmployeesRequest.CreateObj(new() { authorId }, includeDepartments: true));
 
         if (response.Message.IsSuccess)
         {
+          _logger.LogInformation("Department was taken from the service. Department ids: {authorId}", string.Join(", ", authorId));
```
Comment (MariyaSin) [inline src/ProjectService.Business/Commands/Task/GetTaskCommand.cs:77]: not list

File: src/ProjectService.Business/Commands/Project/GetProjectCommand.cs (hunk 2 above)
Comment (VLiamin) [reply src/ProjectService.Business/Commands/Project/GetProjectCommand.cs]: done

File: src/ProjectService.Business/Commands/Project/GetProjectCommand.cs (hunk 3 above)
Comment (VLiamin) [reply src/ProjectService.Business/Commands/Project/GetProjectCommand.cs]: done

File: src/ProjectService.Business/Commands/Task/GetTaskCommand.cs [hunk 5]
```diff
@@ -54,6 +54,8 @@ private async Task<DepartmentData> GetDepartment(Guid authorId, List<string> err
 
       if (departmentFromCache.HasValue)
       {
+        _logger.LogInformation("Department was taken from the cache. Department ids: {authorId}", string.Join(", ", authorId));
```
[the remark this answers is not in the document]
Comment (VLiamin) [reply src/ProjectService.Business/Commands/Task/GetTaskCommand.cs]: done

**Pull request merged**